libraries/standard/mqtt/include/mqtt_state.h

```diff
@@ -19,29 +19,44 @@
  * CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
  */
 
+/**
+ * @file mqtt_state.h
+ * @brief User-facing functions for keeping state of MQTT 3.1.1 packets.
```

Comment:
Have we agreed that all of these functions are meant to be user-facing? Also I'd rephrase to something like "keeping state of MQTT publish deliveries", since it's the publishes we're keeping state of, not the packets themselves.

Reply:
No only the MQTT_PublishToResend it public. I will reword this thanks.